ksql-functional-tests/src/test/java/io/confluent/ksql/test/loader/ExpectedTopologiesTestLoader.java

```diff
@@ -287,8 +290,8 @@ private static TopologyAndConfigs readTopologyFile(
 
   private static String formatQueryName(final String originalQueryName) {
     return originalQueryName
-        .replaceAll(" - (AVRO|JSON)$", "")
-        .replaceAll("\\s|/", "_");
+        .replaceAll(" - (AVRO|JSON|DELIMITED|KAFKA)$", "")
+        .replaceAll(INVALID_FILENAME_CHARS_PATTERN, "_");
```

Comment:
This is the fix for the invalid characters in the file name of expected topologies when checking out on MS Windows.